README.md

```diff
@@ -238,6 +259,24 @@ Log.Logger = new LoggerConfiguration()
 For an example showing how to produce JSON with `ExpressionTemplate`, see the implementation of `ZipkinSink` in this repository,
 and [this article introducing _Serilog.Expressions_ JSON support](https://nblumhardt.com/2021/06/customize-serilog-json-output/).
 
+## Sampling
+
+Sampling is a method of reducing stored data volumes by selectively recording traces. This is similar to levelling, but instead of turning individual span types on and off, sampling causes either _all_ of the spans in a trace to be recorded, or _none_ of them.
```

Comment:
> sampling causes either _all_ of the spans in a trace to be recorded, or _none_ of them.

Does sampling affect log events emitted in an unsampled trace? If so, what happens to them?

Reply:
Note added 👍